### PR TITLE
add subcase to test case vcpu_affinity

### PR DESCRIPTION
**Pull request by brucepatricklee**
set vcpu affinity for none exists vcpu and check xml result

Signed-off-by: Jin Li <jil@redhat.com>

### CODE REVIEW

File: libvirt/tests/src/cpu/vcpu_affinity.py [hunk 1]
```diff
@@ -168,6 +178,9 @@ def check_vcpu_affinity():
                 err_msg = err_msg.split(";")
             libvirt.check_result(result_to_check, err_msg)
 
+    except xcepts.LibvirtXMLError:
+        pass
+
```
Comment (kylazhang) [inline libvirt/tests/src/cpu/vcpu_affinity.py]: Please check this part for exception catch as we discussed.

**Comment by brucepatricklee** [comment]
@kylazhang code updated with trying except xcepts.LibvirtXMLError in  check_vcpu_affinity(). And I want to add some explanation about exception xcepts.LibvirtXMLError. We can the definition below:
class LibvirtXMLError(Exception):

    """
    Error originating within libvirt_xml module
    """

Therefore, it's safe to use xcepts.LibvirtXMLError here as only libvirt_xml module will send this kind of exception.

**Comment by brucepatricklee** [comment]
@chloerh Please help to review, thanks

**Comment by kylazhang** [comment]
OK, got it. Thanks

**Pull request merged**